### DISCUSSION TITLE
Single key gets stuck until it gets pressed again

### DISCUSSION THREAD

**Discussion by chicco-carone** [ppy/osu#29891] [Q&A] [answered]
### Type

Game behaviour

### Bug description

I was just playing normally and then i randomly failed a map and after playing another one and looking at the key counter i notieced the issue. The issue persists after restarting the game.
One key gets stuck after a while and the only way to unstuck it is to press it a bunch of times. I already checked the keyboard and tried another one and also checked and it not an issue outside osu.

### Screenshots or videos

https://github.com/user-attachments/assets/f40d6535-06f9-4c3c-b5f1-7dd32cc070f2

### Version

2024.906.2-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17017999/compressed-logs.zip)

**Comment by smoogipoo** [comment, marked as answer]
It looks like you're using a tablet. Try disabling mouse buttons by pressing F10 in gameplay or via the options.

**Reply by chicco-carone** [reply]
Just checked and noticed that was the issue. I wasn't sure because the input would start to appear a bit after the map started and didn't notice i accidentally pressed the key for mouse input. If i remember correctly on stable there was a notification that said that you have disabled mouse keys which is now missing. I think that since now the tablet settings are into the game directly it should have a separate toggle for tablet input keys which could be by default set to off since the majority of people who play with tablet don't want input from it.
